soh/soh/SohMenuBar.cpp

```diff
@@ -956,6 +956,8 @@ void DrawEnhancementsMenu() {
                                 "This might affect other decal effects\n");
             UIWidgets::PaddedEnhancementSliderInt("Text Spacing: %d", "##TEXTSPACING", "gTextSpacing", 4, 6, "", 6, true, true, true);
             UIWidgets::Tooltip("Space between text characters (useful for HD font textures)");
+            UIWidgets::PaddedEnhancementCheckbox("More info in file select", "gFileSelectMoreInfo", true, false);
```

Comment:
Even though I'm still not a fan of it's current location, the randomizer enhancements submenu is probably better for this. It should also be enabled by default similar to the other randomizer enhancements

Reply:
It's not a randomizer enhancemet specifically, it works with vanilla and mq save files

Reply:
The feature is not limited to randomizer saves, so that may be confusing. Unless we would rather it be forced on for rando saves, and instead the toggle is only used for vanilla/mq saves?

Reply:
I feel like that would also be weird

Reply:
Oh nice, I guess I hadn't considered it being useful outside of rando. disregard my comment then. Default on for rando would be nice to be consistent with zootr